### PR TITLE
cleanup: enable revive(increment-decrement) checking

### PR DESCRIPTION
**Pull request by chaosi-zju**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

cleanup: enable revive(increment-decrement) checking

**Which issue(s) this PR fixes**:

Fixes part of #4490

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by jwcesign — COMMENTED** [review]
/lgtm

File: .golangci.yml [hunk 1]
```diff
@@ -73,6 +73,7 @@ linters-settings:
       - name: error-strings
       - name: error-return
       - name: receiver-naming
+      - name: increment-decrement
```
Comment (RainbowMango) [inline .golangci.yml:76]: @jwcesign have you analyzed how many rules we don't need to fix?
Can we enable them with a single PR?

Comment (jwcesign) [reply .golangci.yml:76]: Around 8.

> Can we enable them with a single PR?

Sure.

Comment (RainbowMango) [reply .golangci.yml:76]: Go for it. It makes no sense for contributors to just add a rule without the fix.
/lgtm
/approve

**Pull request merged**